### ISSUE TITLE
Result Understanding

### ISSUE THREAD

**Issue by khawar-islam** [ispc-lab/LEAD#1]
Hi @sanqingqu,

Thank you for your work. I ran your training script using the command `./scripts/train_source_OPDA.sh`, and I noticed that the dataset folder appears in the checkpoints folder. Inside the checkpoints folder, the OfficeHome dataset is divided into source_0, source_1, source_2, and source_3 folders. How can I match these results with Table 2 in your paper?

What is --s_idx 0, --s_idx 1,--s_idx 2, --s_idx 3? Which results should match with table?

I have also shared the source_0 training log file:
[log_source_training.txt](https://github.com/ispc-lab/LEAD/files/15462703/log_source_training.txt)

**Comment by sanqingqu** [comment]
Hi @khawar-islam,

Thank you for your interest in our work. For simplicity, in our implementation, we set `s_idx` and `t_idx` to denote specific subdomains involved in the dataset. You can refer to the code below for more information about the definitions.

https://github.com/ispc-lab/LEAD/blob/674b08e6a8f2d1a7b9d61d4f8537080d9f398ebb/config/model_config.py#L83-L86

As an example, the OfficeHome dataset contains 4 sub-domains including "Art", "Clipart", "Product" and "Realworld". When setting "s_idx = 0" and "t_idx = 2", it means that we will perform model adaptation from the "Art" domain to the "Product" domain.

**Comment by khawar-islam** [comment]
Hi @sanqingqu 

Thanks for your answer. For instance, in your Table 2. H-Score (%) in OPDA, the LEAD achieves 75.0 in average. Where I can get this number? Like after training, in OfficeHome, there are four folders source_0 to source_3. Could you please tell how i can compute final accuracy?

 **source_0**
```

=================================================
        EVALUATE ON THE TARGET:Clipart                  
=================================================
2024-05-28 09:30:34,884 [INFO] - H-Score:0.474, KnownAcc:0.326, UnknownACC:0.867
2024-05-28 09:30:34,884 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Product                  
=================================================
2024-05-28 09:30:40,372 [INFO] - H-Score:0.711, KnownAcc:0.638, UnknownACC:0.801
2024-05-28 09:30:40,372 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Realworld                  
=================================================
2024-05-28 09:31:21,140 [INFO] - H-Score:0.821, KnownAcc:0.840, UnknownACC:0.804
```

 **source_1**

```
=================================================
        EVALUATE ON THE TARGET:Art                  
=================================================
2024-05-28 09:36:08,483 [INFO] - H-Score:0.534, KnownAcc:0.377, UnknownACC:0.914
2024-05-28 09:36:08,483 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Product                  
=================================================
2024-05-28 09:36:13,749 [INFO] - H-Score:0.566, KnownAcc:0.420, UnknownACC:0.865
2024-05-28 09:36:13,749 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Realworld                  
=================================================
2024-05-28 09:36:34,934 [INFO] - H-Score:0.702, KnownAcc:0.589, UnknownACC:0.869
```

**source_2**

```
=================================================
        EVALUATE ON THE TARGET:Art                  
=================================================
2024-05-28 09:41:16,178 [INFO] - H-Score:0.553, KnownAcc:0.392, UnknownACC:0.934
2024-05-28 09:41:16,178 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Clipart                  
=================================================
2024-05-28 09:41:20,897 [INFO] - H-Score:0.398, KnownAcc:0.255, UnknownACC:0.911
2024-05-28 09:41:20,897 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Realworld                  
=================================================
2024-05-28 09:41:41,389 [INFO] - H-Score:0.766, KnownAcc:0.683, UnknownACC:0.873
```

**source_3**

```
=================================================
        EVALUATE ON THE TARGET:Art                  
=================================================
2024-05-28 09:46:53,913 [INFO] - H-Score:0.593, KnownAcc:0.452, UnknownACC:0.865
2024-05-28 09:46:53,913 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Clipart                  
=================================================
2024-05-28 09:46:58,392 [INFO] - H-Score:0.422, KnownAcc:0.275, UnknownACC:0.899
2024-05-28 09:46:58,392 [INFO] - 
=================================================
        EVALUATE ON THE TARGET:Product                  
=================================================
2024-05-28 09:47:03,228 [INFO] - H-Score:0.733, KnownAcc:0.638, UnknownACC:0.859

```

**Comment by sanqingqu** [comment]
Hi @khawar-islam,

As mentioned in `README.md`, there are two steps to obtain the final model adaptation results. Specifically, the first step is to perform source model pre-training, which you have finished by running `bash ./scripts/train_source_OPDA.sh`. The results are very close to the `source only` in Table 2. To get the model adaptation results by LEAD, you need to run `bash ./scripts/train_target_OPDA.sh`.

**Comment by khawar-islam** [comment]
Hello @sanqingqu thank very much, i understand it.
